src/components/TabsLinks.tsx

```diff
@@ -0,0 +1,101 @@
+import { Link, useLocation } from "react-router-dom";
+import TabsPanels from "./TabsPanels";
+import { useState } from "react";
+
+const TabsLinks = () => {
+  const [navigationTabIndex, setNavigationTabIndex] = useState(0);
+  const [folderTabIndex, setFolderTabIndex] = useState(0);
+  const [inlineTabIndex, setInlineTabIndex] = useState(0);
+
+  const { pathname } = useLocation();
+  const getSelectedIndex = () => {
+    if (pathname.includes("first-tab")) {
+      return 0;
+    } else if (pathname.includes("second-tab")) {
+      return 1;
+    } else if (pathname.includes("third-tab")) {
+      return 2;
+    }
+  };
+
+  return (
+    <div>
+      <h3>Tabs with links.</h3>
+      <p>
+        These tabs are linking to different pages (with different URLs), and
+        this means that you can link to an individual tab via a URL. Try
+        clicking this{" "}
+        <tds-link style={{ display: "inline-block" }}>
+          <a href="/tabs-links/second-tab">link</a>
+        </tds-link>{" "}
+        and you'll land on a page for the second tab.
+      </p>
+      <div className="tds-u-pb3 tabs">
+      <tds-folder-tabs
+        ref={(element) =>
+          element?.addEventListener("tdsChange", (event: any) => {
+            setFolderTabIndex(event.detail.selectedTabIndex);
+            console.log(event);
+          })
+        }
+        mode-variant="secondary" default-selected-index={getSelectedIndex()}>
```

Comment:
I think a simpler way to do this would be something like
```suggestion
        mode-variant="secondary" selected-index={['first-tab', 'second-tab', 'third-tab'].findIndex(pathSegment => pathname.includes(pathSegment))}>
```
Although, this logic would be better put in a useMemo. 
For true realism each page would of course be an actual separate page but not really needed to go that far IMO.